### DISCUSSION TITLE
Get MarkerEntries for all grouped columns not only the one you clicked at

### DISCUSSION THREAD

**Discussion by fax-sma** [patrykandpatrick/vico#335] [Questions] [answered]
Hello everybody,

hopefully someone can help me with that. I want the to display the whole group of columns in a marker of a grouped column chart, but I only get the entry for the directly clicked on column. Does somebody know a way to get the whole group instead?

Thanks in advance

**Comment by Gowsky** [comment, marked as answer]
Hello @fax-sma, unfortunately, this is impossible at the moment. You’d need to update the code in the `ColumnChart`, so [this](https://github.com/patrykandpatrick/vico/blob/0a615bea6c6db5012954094c3407d30c3e11042f/vico/core/src/main/java/com/patrykandpatrick/vico/core/chart/column/ColumnChart.kt#L304) function would need to link other columns in the group to the same x and y coordinates.

**Reply by fax-sma** [reply]
Thank you for answering my question. It did something similar as you suggested, but in the Marker Component.

**Comment by bbetter** [comment]
@fax-sma , any chance you can share what exactly did you do with marker component?

because currently for me as i'm extending CartessianMarker
and as I click on chart
i fall into drawOverlayers and my target contains single element
so I can't build my custom tooltip with multiple entries

**Reply by fax-sma** [reply]
@bbetter 

It's a bit complicated. What we did is we created our own Marker by overriding the `CartesianMarker`. This Marker gets all entries from the Chart and basically when drawing the marker in `drawOverLayers` we not rely on the targets but check the target `x` values with our entries and if needed we draw more then the one entry. But we completely draw our own markers, with custom valueFormatter and shapes. Does that make sense to you?
